HKMP/Game/Client/PlayerManager.cs

```diff
@@ -84,6 +97,118 @@ internal class PlayerManager {
+            new GameObject("Spells") { layer = 9 }.transform.SetParent(playerPrefab.transform);
+            
+            // Copy over mesh filter variables
+            var meshFilter = playerPrefab.GetComponent<MeshFilter>();
+            var mesh = meshFilter.mesh;
+            var localMesh = localPlayerObject.GetComponent<MeshFilter>().sharedMesh;
+            
+            mesh.vertices = localMesh.vertices;
+            mesh.normals = localMesh.normals;
+            mesh.uv = localMesh.uv;
+            mesh.triangles = localMesh.triangles;
+            mesh.tangents = localMesh.tangents;
+
+            // Copy mesh renderer material
+            var meshRenderer = playerPrefab.GetComponent<MeshRenderer>();
+            meshRenderer.material = new Material(localPlayerObject.GetComponent<MeshRenderer>().material);
+            
+            // Disable non bouncer component
+            var nonBouncer = playerPrefab.GetComponent<NonBouncer>();
+            nonBouncer.active = false;
+            
+            // Copy over animation library
+            var spriteAnimator = playerPrefab.GetComponent<tk2dSpriteAnimator>();
+            // Make a smart copy of the sprite animator library so we can
+            // modify the animator without having to worry about other player objects
+            spriteAnimator.Library = CopyUtil.SmartCopySpriteAnimation(
+                localPlayerObject.GetComponent<tk2dSpriteAnimator>().Library,
+                playerPrefab
+            );
+            
+            playerPrefab.transform.SetParent(_playerContainerPrefab.transform);
+            
+            var nameObject = new GameObject("Username");
+            nameObject.transform.position = _playerContainerPrefab.transform.position + Vector3.up * 1.25f;
+            nameObject.name = "Username";
+            nameObject.transform.SetParent(_playerContainerPrefab.transform);
+            nameObject.transform.localScale = new Vector3(0.25f, 0.25f, nameObject.transform.localScale.z);
+            var keepScalePositive = nameObject.AddComponent<KeepWorldScalePositive>();
+            _usernameComponentTypes.Add(keepScalePositive.GetType());
+
+            // Add a TextMeshPro component to it, so we can render text
+            var textMeshObject = nameObject.AddComponent<TextMeshPro>();
+            textMeshObject.text = "Username";
+            textMeshObject.alignment = TextAlignmentOptions.Center;
+            textMeshObject.font = FontManager.InGameNameFont;
+            textMeshObject.fontSize = 22;
+            textMeshObject.outlineWidth = 0.2f;
+            textMeshObject.outlineColor = Color.black;
+            _usernameComponentTypes.Add(textMeshObject.GetType());
+
+            _playerContainerPrefab.SetActive(false);
+            Object.DontDestroyOnLoad(_playerContainerPrefab);
+
+            const int numThreads = 8;
+            var loopsPerThread = InitialPoolSize / numThreads;
```

Comment:
This can be a constant.
```suggestion
            const int loopsPerThread = InitialPoolSize / numThreads;
```